core/ui/StandardLayout.tid

```diff
@@ -18,6 +18,18 @@ code-body: yes
 
 <div class={{{ [all[shadows+tiddlers]tag[$:/tags/ClassFilters/PageTemplate]!is[draft]] :map:flat[subfilter{!!text}] tc-page-container [[tc-page-view-]addsuffix<storyviewTitle>] [[tc-language-]addsuffix<languageTitle>] :and[unique[]join[ ]] }}} >
 
+<$eventcatcher tag="div" class="tc-sidebar-resizer-pointerdown-eventcatcher" selector=".tc-sidebar-resizer" matchSelector=".tc-sidebar-resizer" $pointerdown=<<sidebar-resizer-pointerdown-actions>> $pointerup=<<sidebar-resizer-pointerup-actions>>>
```

Comment:
It's a shame that so much of the resizing logic has to live in the standard layout, it would be better if it could be contained within the tiddler `$:/core/ui/PageTemplate/sidebar`

Reply:
Yes I don't think it necessarily has to. What we need is a place where to put an element that spans the whole page and stays in the background until the sidebar-resizer gets pointer-down'ed - that activates the pointermove div to be on top of everything and the events get "read" and the coordinates get saved in a state tiddler.

Reply:
> It's a shame that so much of the resizing logic has to live in the standard layout, it would be better if it could be contained within the tiddler `$:/core/ui/PageTemplate/sidebar`

What I think now is that it has to be contained within the tiddler `$:/core/ui/PageTemplate/story`. That's because we then can calculate the width of the sidebar based on the `event-fromcatcher-posx` which is the left of the story-river no matter what's on the left of the story river (there can be another PageTemplate or the story-left position can be set to something higher than 0px).
I think like that we can create a div that wraps the story-river and at the right covers the sidebar.